back/config/schemas/settings.schema.json.erb

```diff
@@ -1260,6 +1260,18 @@
         }
       },
 
+      "statistical_insights": {
```

Comment:
@IvaKop I propose we rename this to auto_insights. It's the same term we will use to brand this feature, always good to align as much as we can.

Reply:
Renamed it :)